README.md

```diff
@@ -122,6 +122,172 @@ your container. This will allow you to login to the root account remotely. Local
 connections will still not require a password.
 
 
+Extending the Docker image
+--------------------------
+
+The Docker image provides reasonable set of features that should cover most of the use cases out or the box. For cases when some feature is missing, the image allows consumers to extend the functionality. 
+
+Typically, these changes are: 
+
+* editing other configuration options
+* adjusting how the database is initialized during first start
+* specifying what happens before every start of the database daemon
+* creating more users or databases during database initialization
+
+There are basically two ways how to extend or change the image functionality:
+
+* by creating a new layered Docker image on top of this one, while using this image as base 
+* by volume mounting files during running the Docker image
+
+
+Creating a new layered Docker image on top of this Docker image
+---------------------------------------------------------------
+In order to use this Docker image as base in other layered Docker image, specify name of this one as base and then add files or run commands as you need, like this:
+```
+FROM <this-image>
+ADD mystuff /usr/bin/
+RUN /usr/bin/mystuff
```

Comment:
i would mention that the entrypoint of the image will remain unchanged if you perform these changes